aws/common-io-basic
# Update CI-CD for new checks

Description
-----------
Fix the Typos in this repo, use v3 checkout action

Test Steps
-----------
Ran the spell check

Checklist:
----------


- [ X] I have tested my changes. No regression in existing tests.
- [ X] I have modified and/or added unit-tests to cover the code changes in this Pull Request.

Related Issue
-----------


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
